modules/TestUtils.js

```diff
@@ -88,7 +88,7 @@ export const objectContains = (object, value, compareValues) => {
     compareValues = isEqual
 
   return Object.keys(value).every(k => {
-    if (isObject(object[k])) {
+    if (isObject(object[k]) && Object.keys(value[k]).length > 0) {
```

Comment:
I am agree with you. It is a good idea to throw an warning if a consumer pass an empty object to here.
BTW, I think we should use `Object.getOwnPropertySymbols()` instead of `Object.keys`, because we would also like to compare all symbol properties and non-enumerable properties.